Tests/swift-sdk-swift-tests/DeeplinkTests.swift

```diff
@@ -35,7 +35,9 @@ class DeeplinkTests: XCTestCase {
         
         setupRedirectStubResponse(location: redirectLocation, campaignId: campaignId, templateId: templateId, messageId: messageId)
         
-        IterableAPI.getAndTrack(deeplink: URL(string: iterableRewriteURL)!) { redirectUrl in
+        IterableAPI.handle(universalLink: URL(string: iterableRewriteURL)!)
```

Comment:
Shouldn't handle be inside of getAndTrack? Since handle should use the redirectUrl.

Reply:
`handle`... and `getAndTrack..` are two different methods doing the same thing.
What handle does is it gets the link inside the deep link from the server and calls `IterableURLDelegate` on this. 
What getAndTrack does is it gets the link inside the deep link and then handles the link itself.

That is the reason we are deprecating `getAndTrack` and asking users to move to `handle` instead